modules/local/seqkit/batch_rename/main.nf

```diff
@@ -1,4 +1,4 @@
-process SEQKIT_REPLACE {
+process SEQKIT_BATCH_RENAME {
```

Comment:
We are missing the associated `environment.yml` here, so notihng will be loaded if someone requests `conda`